app/controllers/sites/emails_controller.rb

```diff
@@ -22,11 +22,12 @@ def update
       end
     end
 
-    flash[:notice] = if not_saved
-                       'There were errors saving some of your emails. Please review the marked fields.'
-                     else
-                       'Your support emails have been updated.'
-                     end
+    if not_saved
+      flash.now[:warning] = t('.warning')
```

Comment:
Why is it `warning` here? almost all other errors use `danger`...

Reply:
Since we have 5 flavors of alerts I thought it would fit better warning in this case. Personal choice though.